mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -165,3 +167,51 @@ table.mtp-table--review {
 .mtp-check-header__assigned-user {
   color: $red;
 }
+
+.mtp-check-cell__added-to-my-list::before {
+	content: "✓";
+	font-size: 1.5em;
+	color: #00703c;
+}
+
+.govuk-button {
```

Comment:
you just need to use the `button` class

Reply:
As Igor says, expecially since this our app has a different :root element font size, it makes the text in the buttons too small.